variables.tf

```diff
@@ -124,3 +124,8 @@ variable "cache_behavior" {
   default     = null
 }
 
+variable "custom_header" {
```

Comment:
`custom_header` can be configured on individual origin's level, so it should be part of `var.origin` and not a separate variable.

Please update the code.

Reply:
@antonbabenko thanks for the review. Sorry I missed that. 
Updated. PTAL.